lib/crawlerFactory.js

```diff
@@ -60,6 +60,7 @@ class CrawlerFactory {
         pollingDelay: 5000,
         processingTtl: 60 * 1000,
         promiseTrace: false,
+        retryDelay: 30000,
```

Comment:
great adding it here.  Do recall that it will not be added to the redis based settings automatically.  No need to right now as you have the || 30000 in the code.  Just reminding as I always forget.